demos/mixer-config-quota-bookinfo.yaml

```diff
@@ -22,6 +22,50 @@ data:
         params:
       - name: default
         impl: denyChecker
+    attributes:
```

Comment:
Why do we have all these different configs describing the attributes? Shouldn't the proxy export a set of attributes and the mixer just import that?